docker/java/spring/Dockerfile

```diff
@@ -17,6 +18,7 @@ RUN cd /agent/apm-agent-java \
   && git checkout ${JAVA_AGENT_BRANCH}
 
 COPY maven-package.sh /agent
+COPY .m2 /.m2
```

Comment:
if the folder does not exist it fails

Reply:
I missed to copy the folder. Besides, what do you think? is this a sensible approach?